### PR TITLE
Clear on backspace

### PR DESCRIPTION
**Pull request by AaronBuxbaum**

### CODE REVIEW

File: packages/react-widgets/src/scss/widget.scss [hunk 1]
```diff
@@ -58,6 +58,10 @@
   justify-content: $picker-btn-justify-content;
   padding-left: $picker-btn-padding-left;
   padding-right: $picker-btn-padding-right;
+
+  &:not(:last-child) {
```
Comment (AaronBuxbaum) [inline packages/react-widgets/src/scss/widget.scss:62]: not loving the way this looks... figured it was better than creating different CSS classes for all different possible combinations of buttons, but would be up for a better idea

File: packages/react-widgets/src/DropdownList.tsx [hunk 2]
```diff
@@ -391,9 +401,17 @@ const DropdownListImpl: DropdownList = React.forwardRef(function DropdownList<
...
 
       list.focus(list.prev())
+    } else if (key === 'Backspace' && allowClear && currentValue && !currentSearch) {
+      handleClear(e);
     }
   }
 
+  const handleClear = (e: React.MouseEvent<HTMLSpanElement> | React.KeyboardEvent<HTMLDivElement>) => {
+    e.preventDefault();
+    e.stopPropagation();
```
Comment (jquense) [inline packages/react-widgets/src/DropdownList.tsx:411]: is this to prevent toggling the dropdown open as well?

Comment (AaronBuxbaum) [reply packages/react-widgets/src/DropdownList.tsx:411]: exactly, yes

File: packages/react-widgets/src/scss/widget.scss (hunk 1 above)
Comment (jquense) [reply packages/react-widgets/src/scss/widget.scss:62]: I would add a dropdown specific class for the wrapping div (to remove the inline flex style), you could handle this there.

**Pull request closed without merging**